### PR TITLE
Feature/vega 4588

### PR DESCRIPTION
**Pull request by SergeyKatugin**
## Problem statement
https://artcpt.atlassian.net/browse/VEGA-4758
https://artcpt.atlassian.net/browse/VEGA-4531
https://artcpt.atlassian.net/browse/VEGA-4540
https://artcpt.atlassian.net/browse/VEGA-4530
https://artcpt.atlassian.net/browse/VEGA-4539
https://artcpt.atlassian.net/browse/VEGA-4529
https://artcpt.atlassian.net/browse/VEGA-4538
https://artcpt.atlassian.net/browse/VEGA-4528
https://artcpt.atlassian.net/browse/VEGA-4537
https://artcpt.atlassian.net/browse/VEGA-4534
https://artcpt.atlassian.net/browse/VEGA-4532

## Solution details
Добавление концепций в rb. Полностью добавленные концепции

## Type

- [x] Feature
- [x] Bug Fix
- [x] Refactoring
- [ ] Test
- [ ] Other

## Associated issues

_{Jira issues or PRs related to this one}_

## Quality control

- [x] PR: Based on a correct branch
- [x] PR: Head branch is rebased on the base branch
- [x] PR: Assignee chosen and necessary labels are set
- [x] PR: Current form is filled out (where it makes sense)
- [x] PR: Diff checked for irrelevant changes
- [x] PR: Commit messages follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specs
- [x] PR: Checked for spelling, grammar and typos
- [x] JS: There's no errors/warnings in the browser dev console
- [] Layout: Tested with various content amounts
- [ ] Docs: All the important changes and features are described
- [ ] Tests: Configuration files are checked at test repositories
- [ ] Tests: New unit tests developed
- [ ] Tests: New E2E tests developed
- [ ] Tests: Existing unit tests passed successfully
- [ ] Tests: Existing E2E tests passed successfully
- [x] Tests: Manually tested on personal instance

## Notable statements
- [ ] Affects configuration files
- [ ] Brings new packages or updates existing
- [ ] Opened follow-up tasks to resolve

### CODE REVIEW

File: src/components/CalculateSettings/CalculateSettings.css [hunk 1]
```diff
@@ -53,3 +53,7 @@
     margin-right: var(--space-xs);
   }
 }
+
+.BaseModal__Body {
+  
+}
```
Comment (ndrzhn) [inline src/components/CalculateSettings/CalculateSettings.css:59]: пустой класс?

File: src/pages/Scheme/components/CalculateButton/__tests__/CalculateButton.test.tsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ const renderComponent = (): RenderResult =>
 describe('CalculateButton', () => {
   const getCalculationResultFileIdSpy = jest.spyOn(projectService, 'getCalculationResultFileId');
 
-  test('Срабатывает запуск расчета по нажатию на кнопку "Рассчитать"', async () => {
+  test('Срабатывает запуск расчета по нажатию на кнопку "Рассчитать 1"', async () => {
```
Comment (ndrzhn) [inline src/pages/Scheme/components/CalculateButton/__tests__/CalculateButton.test.tsx:39]: не поняла, для чего 1

Comment (SergeyKatugin) [reply src/pages/Scheme/components/CalculateButton/__tests__/CalculateButton.test.tsx:39]: похоже лишнее

File: src/components/CalculateSettings/CalculateSettings.css (hunk 1 above)
Comment (SergeyKatugin) [reply src/components/CalculateSettings/CalculateSettings.css:59]: убрал

**Review by ndrzhn — DISMISSED** [review]

**Review by ndrzhn — APPROVED** [review]

**Pull request merged**